tools/cjxl_main.cc

```diff
@@ -144,7 +144,9 @@ struct CompressArgs {
     cmdline->AddOptionValue(
         'e', "effort", "EFFORT",
         "Encoder effort setting. Range: 1 .. 9.\n"
-        "    Default: 7. Higher number is more effort (slower).",
+        "Default: 7. Higher numbers allow more computation, generally "
```

Comment:
Please check indentation and line breaks. Aiming to stay below 100 characters width and indentation consistent (the first line of the description automatically get indented 4 spaces so any additional lines also need 4 spaces).

Reply:
I know the lines are still long, but should be within what you asked. Also fixed the group ordering descriptions so they matched all the other formatting